### PR TITLE
Fix terraform validation issues

### PR DESCRIPTION
**Pull request by zackproser**
Fixes for the minor issues preventing a few modules from passing `terraform validate` as uncovered by the new `TestValidateAllTerraformModulesOnTerratest` test.

As can be seen in[ this test run](https://circleci.com/api/v1.1/project/github/gruntwork-io/terratest/5871/output/106/0?file=true&allocation-id=60ca4ed611a6f444d04271a5-0-build%2F2F80911B), all `TestValidateAllTerraformModulesOnTerratest` sub tests are now passing.

### CODE REVIEW

File: test/terraform_aws_s3_example_test.go [hunk 1]
```diff
@@ -36,6 +36,8 @@ func TestTerraformAwsS3Example(t *testing.T) {
 			"tag_bucket_name":        expectedName,
 			"tag_bucket_environment": expectedEnvironment,
 			"with_policy":            "true",
+			// Passing region to Terraform here is required for the terraform validate command to pass
+			"region": awsRegion,
```
Comment (yorinasub17) [inline test/terraform_aws_s3_example_test.go:40]: If we're now setting the region this way, we should remove the `AWS_DEFAULT_REGION` env var configuration down below.

**Review by yorinasub17 — COMMENTED** [review]
Had one more consistency fix to suggest. Once that is addressed, and the build is passing, this is good to go!

File: examples/terraform-aws-ecs-example/main.tf [hunk 2]
```diff
@@ -2,6 +2,9 @@
 # PIN TERRAFORM VERSION TO >= 0.12
 # The examples have been upgraded to 0.12 syntax
 # ---------------------------------------------------------------------------------------------------------------------
+provider "aws" {
+  region = "us-east-1"
```
Comment (rhoboat) [inline examples/terraform-aws-ecs-example/main.tf:6]: Did we want to make this a var instead of hardcoded string?

Comment (zackproser) [reply examples/terraform-aws-ecs-example/main.tf:6]: Yes - thanks!

File: test/terraform_aws_s3_example_test.go (hunk 1 above)
Comment (zackproser) [reply test/terraform_aws_s3_example_test.go]: Fixed in f5c6a3f

File: examples/terraform-aws-ecs-example/main.tf (hunk 2 above)
Comment (zackproser) [reply examples/terraform-aws-ecs-example/main.tf]: Fixed in f5c6a3f

**Review by yorinasub17 — APPROVED** [review]
LGTM, assuming the build passes!

**Comment by zackproser** [comment]
> LGTM, assuming the build passes!

Thanks both for the reviews!

**Review by yorinasub17 — APPROVED** [review]
Approving, assuming build passes.

**Review by rhoboat — APPROVED** [review]

**Review by brikis98 — APPROVED** [review]

**Pull request merged**